### ISSUE TITLE
contrast - book nic link - speaking

### ISSUE THREAD

**Issue by vavroom** [nicolas-steenhout/main#96]
◦	The “Book Nic” link in the “Keynote and presentation topics” section has insufficient contrast in light mode.

**Comment by vavroom** [comment]
Will be taken care of with #77
